### ISSUE TITLE
Folder figures does not exist

### ISSUE THREAD

**Issue by LuisYordano** [PacktPublishing/Practical-Computer-Vision#2]
In all the chapters reference this folder ../figures/.

And in chapter 8 - file: "08_compute_F_mat.py", there is a path (line 81):
----------------------
image_dir = '/ Users / mac / Documents / dinoRing /'
---------------------

Please indicate where to download these files from the book.

**Comment by richard5635** [comment]
Yes, I am currently looking for the folder as well. Please help indicate the download location.
